modules/nf-core/limma/differential/environment.yml

```diff
@@ -1,5 +1,9 @@
 channels:
-  - conda-forge
-  - bioconda
+- conda-forge
+- bioconda
 dependencies:
-  - bioconda::bioconductor-limma=3.54.0
+- bioconda::bioconductor-edger=4.0.16
+- bioconda::bioconductor-ihw=1.28.0
+- bioconda::bioconductor-limma=3.58.1
```

Comment:
Limma is a [dependency of edger](https://github.com/bioconda/bioconda-recipes/blob/a489320df2777dfed69d979db18af47bcaa8a2d5/recipes/bioconductor-edger/meta.yaml#L24), you don't need to include it here

Reply:
We only pin the primary dependency in the modules repo, unless you have a really compelling reason to do otherwise. We will at some point have lock files that pin all dependencies.

Reply:
(as in other comments, I'm not sure we need edgeR here, so we'd only pin Limma)

Reply:
Well we noticed that results are slightly different depending on limma versions so we decided to pin it to achieve reproducibility. I will adopt your comment in standalone module

Reply:
Actually, I see your point on this one, apologies. Limma is the primary dependency so it's pinned. It's edgeR we probably don't need to bother pinning (though we need to include it).